.github/CODEOWNERS

```diff
@@ -15,10 +18,11 @@
 /hbase_master/          @everpeace
 /hbase_regionserver/    @everpeace
 /launchdarkly/          support@launchdarkly.com
-/lighthouse/		@ericmustin
+/lighthouse/            @ericmustin
```

Comment:
what's up here?

Reply:
I think these had tabs instead of spaces and threw the formatting off